scripts/run_mypy_checks.py

```diff
@@ -586,6 +586,10 @@
     'extensions/visualizations/models.py',
     'jobs/base_jobs.py',
     'jobs/base_jobs_test.py',
+    'jobs/blog_validation_errors.py',
+    'jobs/blog_validation_errors_test.py',
+    'jobs/blog_validation_jobs.py',
+    'jobs/blog_validation_jobs_test.py',
```

Comment:
@hardikkat24 Are we expecting to add newfiles in this list?

Reply:
It should be avoided to add more files in the denylist. @Rijuta-s Can you please add type annotations? Thanks!

Reply:
Done